packages/neutron-sdk/src/interchain_queries/v045/types.rs

```diff
@@ -42,6 +43,10 @@ pub const DELEGATION_KEY: u8 = 0x31;
 /// <https://github.com/cosmos/cosmos-sdk/blob/35ae2c4c72d4aeb33447d5a7af23ca47f786606e/x/staking/types/keys.go#L35>
 pub const VALIDATORS_KEY: u8 = 0x21;
 
+/// Key for validators in the **staking** module's storage
+/// <https://github.com/cosmos/cosmos-sdk/blob/35ae2c4c72d4aeb33447d5a7af23ca47f786606e/x/slashing/types/keys.go#L34>
+pub const VALIDATOR_SIGNING_INFO_KEY: u8 = 0x01;
```

Comment:
```suggestion
/// Key for validators in the **slashing** module's storage
/// <https://github.com/cosmos/cosmos-sdk/blob/35ae2c4c72d4aeb33447d5a7af23ca47f786606e/x/slashing/types/keys.go#L34>
pub const VALIDATOR_SIGNING_INFO_KEY: u8 = 0x01;
```

Reply:
FYI: it is already fixed in #124, so if we merge it, you wouldn't have to fix it here